ci/build_container/build_recipes/boringssl.sh

```diff
@@ -1,10 +1,16 @@
+#!/bin/bash
```

Comment:
Why add this, it's only used when testing during development, in which case we can just run under `bash`. Same goes for making them executable.